fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1688,13 +1687,26 @@ public class Config extends ConfigBase {
     public static String jaeger_grpc_endpoint = "";
 
     @ConfField
-    public static long experimental_lake_compaction_max_version_count = 10;
+    public static String lake_compaction_selector = "com.starrocks.lake.compaction.SimpleSelector";
```

Comment:
I think use the package + class as config value is not good, because the package or class name may be changed in the future.

Reply:
updated